README.md

```diff
@@ -1 +1,3 @@
-# north-star-team7
\ No newline at end of file
+# north-star-team7
```

Comment:
⚠️ Regarding the general review of the font, I believe it's preferable to use the font via CDNs or from the internet, rather than using a local font. This approach imporve the appearance of the code. However, if you're unable to locate a specific font you desire online, you are welcome to use it locally.